internal/pkg/aws/ecr/ecr.go

```diff
@@ -122,19 +136,27 @@ func (s Service) DeleteImages(images []Image, repoName string) error {
 		return nil
 	}
 
-	var imageIdentifiers []*ecr.ImageIdentifier
-	for _, image := range images {
-		imageIdentifiers = append(imageIdentifiers, image.imageIdentifier())
+	var imageIdentifiers [][]*ecr.ImageIdentifier
+	for ind, image := range images {
+		if ind%batchDeleteLimit == 0 {
```

Comment:
Does this create an empty "batch" for the very first index?

Reply:
Yeah I think so. What's the problem for that?

Reply:
It's an extra API call with an empty list that will never do any work.

Reply:
It's not an empty list because if there's no image more to add then it jumps out of the loop. It creates an empty list for adding new ones, since this is an array of arrays.

Reply:
Ah I see, you're right. :+1: